tests/test_main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+from app.main import is_isogram
+
+
+def test_for_empty_string():
+    assert is_isogram("") is True
+
+
+def test_case_insensitivity():
```

Comment:
To check this, it is better to check that if a word contains the same letter but in different cases, it should not be considered an isogram.

Reply:
Because if the function is case-sensitive, it will also return the same results for the values you give it.